### ISSUE TITLE
SikuliX error due to OpenJDK v8 64-bit installed but not detected - pending details

### ISSUE THREAD

**Issue by abitamaspp** [tebelorg/RPA-Python#199]
I'm using Jupyter Notebook on Windows 10

I have installed OpenJDK version 8 (Amazon Corretto) and also have followed the installation as written in their page. However, I get this message when trying to use init() with visual_automation = True
![image](https://user-images.githubusercontent.com/56988153/104279749-65f51c00-54dd-11eb-9a65-1984bdc0c29c.png)

I've tried to follow your suggestion (https://github.com/kelaberetiv/TagUI/issues/871#issuecomment-734093588) but I don't understand about '...run `tagui live` from command prompt...'. Can you please help me?

(PS: please stay safe and healthy)

**Comment by abitamaspp** [comment]
I found the solution.. or maybe more appropriately a work-around.

TLDR: uninstall Corretto, then install OpenJDK (download from here -> [https://jdk.java.net/15/](https://jdk.java.net/15/))

When I look at the source code (tagui.py), I found that TagUI will check whether installed JDK is 64-bit or not. It will only check if `visual_automation = True` included in `init()`. Basically the checking process will see whether string '64-bit' is exist or not in the Java version's description. Unfortunately the JDK from Amazon Corretto doesn't has it.

Then I re-read and followed Yu Chen Kuang's guide on [setting up visual automation](https://yuchenkuang.com/TagUI-Python-Setup/setup-tagui-python-on-windows-10-with-visual-automation.php). And yes, you guess it. His guide is telling to install OpenJDK. Then after installing OpenJDK and setting environment variables, I can use `keyboard()` function.

Hope this will help others newbie like me.

**Comment by kensoh** [comment]
Hi Abitama, thanks for sharing back your solution! Reopening issue because this needs a permanent fix.

In the package, it will check for the string '64-bit' or '64 bit' string when `java -version` is run.

When I tested the Amazon Corretto has that string but maybe that has changed now.

Do you remember what was shown on the screen when you do `java -version` manually?

**Comment by kensoh** [comment]
Adding on, for my macOS, the OpenJDK comes back with `64-Bit` response which would be a match in the package since the checking is case-insensitive. So far has not heard this from other users, looking out for more inputs from you and others.

```
RPA:rpa-python kensoh$ java -version
openjdk version "1.8.0_232"
OpenJDK Runtime Environment Corretto-8.232.09.1 (build 1.8.0_232-b09)
OpenJDK 64-Bit Server VM Corretto-8.232.09.1 (build 25.232-b09, mixed mode)
```

**Comment by kensoh** [comment]
Downloading and installing for the macOS OpenJDK link from https://tagui.readthedocs.io/en/latest/setup.html

Still gets 64-bit string in the `java -version`
```
openjdk version "1.8.0_282"
OpenJDK Runtime Environment Corretto-8.282.08.1 (build 1.8.0_282-b08)
OpenJDK 64-Bit Server VM Corretto-8.282.08.1 (build 25.282-b08, mixed mode)
```

**Comment by kensoh** [comment]
Downloading from above installation instructions for Windows OpenJDK link also gets back same output as macOS. The instructions URLs above point to the same as when navigating from https://aws.amazon.com/corretto

If you or other users can tell me more your output when running `java -version` then I can check further. For now, I'll just leave the help message the same. Corretto is much easier to install just setup directly.

**Comment by abitamaspp** [comment]
Hi Ken, I'm sorry for late replying. And also apologize because I don't remember what appear when I typed `java -version` back then (my bad, should have included it)
